ubuntu_18.04-dev.dockerfile

```diff
@@ -0,0 +1,107 @@
+FROM ubuntu:18.04
+
+# Ubuntu Setup
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+RUN apt-get update \
+    && apt-get install -y --fix-missing \
+            software-properties-common wget g++ \
+            build-essential python-numpy python-scipy cython python-setuptools \
+            python-nose git cmake vim emacs gfortran libblas-dev \
+            liblapack-dev libhdf5-dev libhdf5-serial-dev gfortran python-tables \
+            python-matplotlib python-jinja2 python-dev libpython-dev \
+            autoconf libtool python-setuptools python-pip doxygen \
+    && apt-get clean -y \
+    && pip install --force-reinstall \
+            sphinx \
+            cloud_sptheme \
+            prettytable \
+            sphinxcontrib_bibtex \
+            numpydoc \
+            nbconvert \
+            numpy \
+            cython
+
+# Script conditional setup: Default PyNE alone
+ARG build_moab=NO
+ARG enable_pymoab=NO
+ARG build_dagmc=NO
```

Comment:
I think these should be default "yes". Our current install scripts build MOAB by default. I see no reason to not have that set as the default to give users the full capability. These scripts should be meant to allow new users the to build PyNE with all of it's capabilities with little effort or knowledge of its dependencies.

Reply:
I was thinking that PyNE users necessary don't want the DAGMC/MOAB/pyMOAB capabilities.
and so it is simpler to add them that remove them..

I don't have a strong opinion there, I have the feeling those option will mostly be used to create the image required to run the CI....